ui/app/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -150,58 +157,68 @@ export default class PermissionConnect extends Component {
       permissionsRequest,
       addressLastConnectedMap,
       permissionsRequestId,
+      connectPath,
+      confirmPermissionPath,
+      page,
     } = this.props
-    const { page, selectedAccountAddress, permissionAccepted, originName } = this.state
+    const { selectedAccountAddress, permissionAccepted, originName, redirecting } = this.state
 
     return (
       <div className="permissions-connect">
         { page !== null
```

Comment:
```suggestion
        { !redirecting
```
Presumably this should be updated, as `page` should never be set to  `null` anymore